start.py

```diff
@@ -1,40 +1,100 @@
-from board import Board, EndStates, SpotStates, BoardState, WinningCombos 
+from board import Board, EndStates, SpotStates, BoardState, WinningCombos
 from user import User, UserActions
+from ai import Ai
 import board
 from ui import Ui, BoardPresenter, CommandLinePrompt
 
-def start_game(current_state, player1, player2):
-    Ui.msg(BoardPresenter.display_terminal_board(current_state))
-    
-    if EndStates.is_draw(current_state):
-        Ui.msg('DRAW. GameOver')
-        return True
-     
-    response = CommandLinePrompt.get_input("Enter a number from 1-9: ") 
-    response = UserActions.make_move(current_state, response)
-     
-    if response == False:
-        start_game(current_state, player1, player2)
-
-    # update the board state
-    BoardState.update_state(current_state, User.current_player, response)
-     
-    if EndStates.did_a_player_win(current_state, User.current_player, win_config.winning_combos):
-        Ui.msg(BoardPresenter.display_terminal_board(current_state)) 
-        Ui.msg('Game Over: ' + User.current_player + ' WINS!')
-        return True
-
-    if User.current_player == player1:
-        User.switch_current_user(User.current_player, player2)
-    else:
-        User.switch_current_user(User.current_player, player1)
-    start_game(current_state, player1, player2)
+class GameModes:
+    def human_v_human(current_state, player1, player2):
+        game_over = False
+        while game_over is False:
+            response = False
+            while response is False:
+                Ui.msg(BoardPresenter.display_terminal_board(current_state))
+                response = CommandLinePrompt.get_input("Enter a number from 1-9: ")
+                response = UserActions.make_move(current_state, response)
+            BoardState.update_state(current_state, User.current_player, response)
+
+            if EndStates.did_a_player_win(current_state, User.current_player, win_config.winning_combos):
+                Ui.msg(BoardPresenter.display_terminal_board(current_state))
+                Ui.msg('Game Over: ' + User.current_player + ' WINS!')
+                return True
+
+            if EndStates.is_draw(current_state):
+                Ui.msg('DRAW. GameOver')
+                return True
+
+            if User.current_player == player1:
+                User.switch_current_user(User.current_player, player2)
```

Comment:
I may have said this last time, but are you sure this is the User's responsibility?